### PR TITLE
doc(ai): sync/merge the ai documentation with README

### PR DESCRIPTION
**Pull request by returnToInnocence**
## Purpose of the PR

- Merge the [official documentation](https://hugegraph.apache.org/cn/docs/quickstart/hugegraph-ai/) and the project [readme](https://github.com/apache/incubator-hugegraph-ai/blob/main/hugegraph-llm/README.md)

### CODE REVIEW

File: content/cn/docs/quickstart/hugegraph-ai.md [hunk 1]
```diff
@@ -1,148 +1,177 @@
...
----
 
 ### 1 HugeGraph-AI 概述
 hugegraph-ai 旨在探索 HugeGraph 与人工智能（AI）的融合，包括与大模型结合的应用，与图机器学习组件的集成等，为开发者在项目中利用 HugeGraph
 的 AI 能力提供全面支持。
 
 ### 2 环境要求
-- python 3.8+
-- hugegraph 1.0.0+ 
+- python 3.9+
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-ai.md:10]:
```suggestion
- python 3.9+  (better to use `3.10`)
```

File: content/cn/docs/quickstart/hugegraph-ai.md [hunk 2]
```diff
@@ -1,148 +1,177 @@
...
 
 ### 1 HugeGraph-AI 概述
 hugegraph-ai 旨在探索 HugeGraph 与人工智能（AI）的融合，包括与大模型结合的应用，与图机器学习组件的集成等，为开发者在项目中利用 HugeGraph
 的 AI 能力提供全面支持。
 
 ### 2 环境要求
-- python 3.8+
-- hugegraph 1.0.0+ 
+- python 3.9+ 
+- hugegraph-server 1.2+
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-ai.md:11]:
```suggestion
- hugegraph-server 1.3+
```

File: content/cn/docs/quickstart/hugegraph-ai.md [hunk 3]
```diff
@@ -1,148 +1,177 @@
...
-  - Text: 输入的文本。 
-  - Schema：接受以下两种类型的文本： 
-    - 用户定义的 JSON 格式模式。 
-    - 指定 HugeGraph 图实例的名称，它将自动提取图的模式。
-  - Disambiguate word sense：是否进行词义消除歧义。 
-  - Commit to hugegraph：是否将构建的知识图谱提交到 HugeGraph 服务器
-
-![gradio 配置](/docs/images/gradio-kg.png)
+
+1. 启动HugeGraph数据库，可以通过[Docker](https://hub.docker.com/r/hugegraph/hugegraph) / [Binary Package](https://hugegraph.apache.org/docs/download/download/)运行它。
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-ai.md:15]: 注意中英文空格, 使用插件/工具格式化一下 (参考 https://github.com/apache/incubator-hugegraph-doc/pull/282#issuecomment-1719156940)

**Review by imbajin — COMMENTED** [review]
先确认 CN 正常后, 再一次性同步 EN 避免多次变更

File: content/cn/docs/quickstart/hugegraph-ai.md (hunk 3 above)
Comment (returnToInnocence) [reply content/cn/docs/quickstart/hugegraph-ai.md:15]: get

File: content/cn/docs/quickstart/hugegraph-ai.md [hunk 4]
```diff
@@ -9,140 +9,154 @@ hugegraph-ai 旨在探索 HugeGraph 与人工智能（AI）的融合，包括与
...
+    ```
+5. 启动 **Graph RAG** 的 gradio 交互 demo，可以使用以下命令运行，启动后打开 http://127.0.0.1:8001
+    ```bash
+    python3 -m hugegraph_llm.demo.rag_demo.app
+    ```
+    默认主机为 `0.0.0.0` ，端口为 `8001` 。您可以通过传递命令行参数 `--host` 和 `--port` 来更改它们。
+    ```bash
+    python3 -m hugegraph_llm.demo.rag_demo.app --host 127.0.0.1 --port 18001
+    ```
+6. 启动 **Text2Gremlin** 的 gradio 交互演示，可以使用以下命令运行，启动后打开 http://127.0.0.1:8002。您还可以按上述方式更改默认主机 `0.0.0.0` 和端口 `8002` 。(🚧ing)
```
Comment (imbajin) [inline content/cn/docs/quickstart/hugegraph-ai.md:41]: fix the output (preview)

**Review by imbajin — APPROVED** [review]

**Pull request merged**